rpqueue/__init__.py

```diff
@@ -153,6 +152,42 @@ def set_key_prefix(pfix):
 logging.basicConfig()
 log_handler = logging.root
 
+# Case-insensitive set of key names to redact from logs
+SENSITIVE_REDACT_KEYS = frozenset({
+    'api_key', '$api_key', 'apikey', 'api-key', 'x-api-key', 'password',
+    'secret', 'secret_key', 'client_secret', 'secret-key', 'secret_access_key', 'secret-access-key',
+    'token', 'access_token', 'refresh_token', 'id_token', 'auth_token', 'oauth_token',
+})
```

Comment:
[non-blocking suggestion] wondering if it's too wide of a net to cast if we should consider leveraging regex to account for any string containing the following substrings? That way we don't have to update this list every time there's something new to be filtered.
```
regex = r".*?(token|key|access|password|auth).*?"
pattern = re.compile(regex, re.IGNORECASE)
pattern.search(k_str)
```